src/adapters/MeetingsJSONAdapter/controls/SwitchCameraControl.js

```diff
@@ -43,6 +43,8 @@ export default class SwitchCameraControl extends MeetingControl {
           noOptionsMessage: 'No available cameras',
           options: null,
           selected: null,
+          type: 'DROPDOWN',
```

Comment:
please update this pr to look at the control type for:
- making a button green
- making the leave button red
- displaying a dropdown instead of a button
- other applicable decisions (if any)